test-app/tests/test-helper.ts

```diff
@@ -7,10 +7,10 @@ import { start } from 'ember-qunit';
 import setupSinon from 'ember-sinon-qunit';
 import { setRunOptions } from 'ember-a11y-testing/test-support';
 
-setApplication(Application.create(config.APP));
-
 setup(QUnit.assert);
 
+setApplication(Application.create(config.APP));
```

Comment:
Was there a reason to change this?

Reply:
I cannot recall any specific reason. :see_no_evil: Good catch. I reverted the change.